src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -1211,7 +1211,7 @@ configurationRegistry.registerConfiguration({
 		[NotebookSetting.cellGenerate]: {
 			markdownDescription: nls.localize('notebook.cellGenerate', "Enable experimental generate action to create code cell with inline chat enabled."),
 			type: 'boolean',
-			default: typeof product.quality === 'string' && product.quality !== 'stable',
+			default: true,
 			tags: ['experimental']
```

Comment:
Is it still experimental?

Reply:
good catch, I removed the experimental tag. Will rename the setting and do the transition next month.